pytorch_toolkit/action_recognition_2/model_templates/gesture_recognition/s3d-rgb-mobilenet-v3-stream-jester/template.yaml

```diff
@@ -0,0 +1,62 @@
+name: s3d-rgb-mobilenet-v3-stream-jester
+domain: Action Recognition 2
+problem: Gesture Recognition
+framework: OTEAction
+summary: ASL Recognition based on S3D-MobileNet-V3.
+annotation_format: CustomAction
+dependencies:
+- sha256: d55d4a208e2a0d379026665d6d5bf7c191b9cc4c52672733df1df878a6b3d7c6
```

Comment:
why are there two snapshots?

Reply:
One is to train from some kind of scratch (pre-trained on general action recognition dataset only) and the rest one - the usual model snapshot for fine-tuning only.

Reply:
I think there must be one of them only and it must have hard-coded 'snapshot.pth' destination. There is no known way how we can use two snapshots. I guess both snapshots might be mentioned in documentation (md) page in **training** and **fine-tuning** sections. You might want to extend table describing models.

Reply:
Ok, accepted.

Reply:
Done.